Lombiq.Hosting.Tenants.Admin/Readme.md

```diff
@@ -0,0 +1,46 @@
+# Lombiq Hosting Tenants - Admin
+
+## About
+
+With the help of this module, you can log in from the Default tenant admin dashboard to any other tenants as an administrator user.
+
+## Documentation
+
+This module contains two features:
+- Lombiq.Hosting.Tenants.Admin
+- Lombiq.Hosting.Tenants.Admin.SubTenant
+
+### Lombiq.Hosting.Tenants.Admin
+
+This feature is only available on the Default tenant. It provides a button on the tenant edit page. After clicking this button the site login to the selected tenant as an administrator and redirect to the tenant's admin dashboard. 
```

Comment:
```suggestion
This feature is only available on the Default tenant. It provides a button on the tenant editor page. After clicking this button, you'll be logged to the selected tenant as an administrator and be redirected to its admin dashboard. 
```

Reply:
Ezután rakj be egy screenshotot a buttonnal légyszi. Ez egy "Demo" nevű új tenant legyen hasonlóan dummy részletekkel. Elég, ha csak a tenant editor látszik. Szal a cél az, hogy a user megértse, hol keresse a buttont.